### PR TITLE
fix: provide UI5 specific configs

### PR DESCRIPTION
**Pull request by flovogt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,9 @@
     "node": ">= 8.5",
     "npm": ">= 5"
   },
+  "dependencies": {
+    "@openui5/sap.ui.core": "1.66.1"
```
Comment (matz3) [inline package.json]: Did you use a fixed version on purpose?
I would recommend to use a range with the minimum version required (e.g. `^1.66.1` or even a lower version, as I think you don't use any of the newer APIs).
But as discussed, this is something that still needs to clarified...

Comment (flovogt) [reply package.json]: You're right. We'll make a downport testing and enter this as the version number

File: package.json [hunk 2]
```diff
@@ -52,6 +57,7 @@
   },
   "files": [
     "src/sap/fhir/**/*.js",
-    "ui5.yaml"
+    "ui5.yaml",
+    "LICENSE.txt"
```
Comment (matz3) [inline package.json]: FYI, LICENSE files are always included: https://docs.npmjs.com/files/package.json#files

Comment (flovogt) [reply package.json]: Thx for this hint.

**Pull request merged**